### ISSUE TITLE
Ability to import/include other DBML

### ISSUE THREAD

**Issue by tv-long** [holistics/dbml#585]
Hi Team,
Our organization applied Data Mesh architecture to our EDW. Which means we have a lot of Data Product maintained by multiple Teams and they are related to each other. Therefore, we would like to have the ability to include or refer some objects from other DBML to avoid duplicating Table Definition everywhere it's used.
For example, we could describe TABLE_C which link to TABLE_A and TABLE_B in another project like this:
```
Include 'project/docs/dbml/dataproduct.dbml' {
    provider: 'gitlab'
    token: '$GITLAB_TOKEN'
    table: [TABLE_A, TABLE_B]
}

Include './internal_dataproduct.dbml' {
    provider: 'file'
    table: [TABLE_D]
}

Table TABLE_C {
    column_a string [ref: - TABLE_A.column]
    column_b string [ref: - TABLE_B.column]
    column_c long
    column_d string [ref: - TABLE_D.column]
}
```
We expected that all relationships between TABLE_A and TABLE_B will be included but none of other tables.

Best regards
Long

**Comment by JabDoesThings** [comment]
Is this still a being looked at as a feature?

**Comment by huydo862003** [comment]
Yes, it's on the roadmap for the next 2 quarters: https://holistics.notion.site/Public-Product-Roadmap-dbx-dbdiagram-dbdocs-dbml-c36af127c86a40919337b61ac6e12a48

**Comment by huydo862003** [comment]
Hi @JabDoesThings, @tv-long, DBML now supports multiple files in version 8.0.0. For now, only dbml-cli and the dbml-playground supports it. We will support it in dbdiagram as a next step

You can check this doc page for more details: https://dbml.dbdiagram.io/syntax/module-system
